tests/yam/agama/agama.pm

```diff
@@ -26,6 +29,15 @@ sub run {
     my $test = get_required_var('AGAMA_TEST');
     my $reboot_page = $testapi::distri->get_reboot();
 
+    if (is_sle() && is_s390x()) {
+        # The ISO does not have SUSE ca-certs, so we need to install them manually.
+        # Record a soft failure for this, when the bug is opened.
+        record_soft_failure("bsc#1231421");
+        assert_script_run("curl --insecure -o /tmp/ca-certificates-suse.rpm https://download.suse.de/browse/ibs/SUSE:/CA/SLE_15_SP7/noarch/ca-certificates-suse-1.0-150700.8.1.noarch.rpm");
+        assert_script_run("zypper -n install /tmp/ca-certificates-suse.rpm");
+        assert_script_run("systemctl restart agama");
+    }
+
```

Comment:
we need the same workaround for unattended installation as well, agama_base might be a place to put it, I would prefer not to have a library...could you please take a look if that would be a good place and run verification for unattended.

Reply:
9f6fb3b18ccbfb0753821bb54e5b3b78455a4e7e